deployments/helm/nvidia-dra-driver-gpu/templates/controller.yaml

```diff
@@ -34,6 +34,7 @@ spec:
       {{- end }}
       labels:
         {{- include "nvidia-dra-driver-gpu.templateLabels" . | nindent 8 }}
```

Comment:
Note that `templateLabels` still includes `selectorLabels`. Is this intended now that `selectorLabels` have been removed from `selector.matchLabels` above?

Reply:
> Note that templateLabels still includes selectorLabels

Yes. That's fine.

I wanted to be minimally invasive. To me, that first and foremost meant that I did not want to remove any labels.

Reply:
another update:

> Note that templateLabels still includes selectorLabels

not anymore, see commentary.

Reply:
Thanks: I think making selectorLabels more specific (and a reduced set is correct).